### PR TITLE
Front-end Development: Added questions 49, 50 and 51

### PR DESCRIPTION
**Pull request by belce1982**
Check question 51, not sure about the answer at all.

I had done the quiz today, 08/11, and I had found some questions that were not in the repo.

Thanks for your hard work.

### CODE REVIEW

**Comment by belce1982** [comment]
@vanekbr
@ShankS3

File: front-end-development/front-end-development-quiz.md [hunk 1]
```diff
@@ -467,3 +467,29 @@ console.log(currencies);
...
+- [ ] `<navigation>`
+- [ ] `<main>`
+- [ ] `<ul>`
+
+[Source: W3Schools](https://www.w3schools.com/tags/tag_nav.asp)
+
+#### Q51. You discover that CSS is being loaded on a website using the following code. Why might a web developer have done this?
+`<link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="style.css"></noscript>`
+
+- [ ] The CSS is being loaded asynchronously in order to prevent render blocking.
```
Comment (vanekbr) [inline front-end-development/front-end-development-quiz.md:492]: This answer is the correct one. Source: https://stackoverflow.com/a/46750893
@belce1982 Please, update the MR 🙏🏻

Comment (belce1982) [reply front-end-development/front-end-development-quiz.md]: Done, thanks!

**Review by vanekbr — APPROVED** [review]

**Comment by vanekbr** [comment]
@Ebazhanov This one can be merged ✔️

**Review by Ebazhanov — APPROVED** [review]
Great job 👍 , thank you @belce1982!

**Comment by Ebazhanov** [comment]
@all-contributors please add @belce1982 for design, code

**Comment by asbillt** [comment]
Can I get y'all to check the pull request I made on Sunday, it has been updated to resolve conflicts. Thank you.

**Pull request merged**